src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.ts

```diff
@@ -115,4 +151,52 @@ export class NvmeofGatewayGroupComponent implements OnInit {
   updateSelection(selection: CdTableSelection): void {
     this.selection = selection;
   }
+
+  deleteGatewayGroupModal() {
+    const selectedGroup = this.selection.first();
+    if (!selectedGroup) {
+      return;
+    }
+    const {
+      service_name: serviceName,
+      spec: { group }
+    } = selectedGroup;
+
+    const disable = selectedGroup.subSystemCount > 0 || !group;
+
+    this.modalService.show(DeleteConfirmationModalComponent, {
+      impact: DeletionImpact.high,
+      itemNames: [group],
+      itemDescription: $localize`gateway group`,
+      bodyTemplate: this.deleteTpl,
+      bodyContext: {
+        groupName: group,
+        subsystemCount: selectedGroup.subSystemCount,
+        disable,
```

Comment:
variable should be named in a way that it makes sense just by looking at it.
```suggestion
        disableForm,
```

Reply:
changes are made